### DISCUSSION TITLE
No upload button on application

### DISCUSSION THREAD

**Discussion by favarogabriel** [community/community#131555] [GitHub Education]
### Hi everyone,

When trying to sign up for Github's student program, I ran into a very serious problem. 

It's simply not possible to attach a file to prove that I'm enrolled or anything like that. Furthermore, even using my cell phone, I cannot use the rear camera on my phone, and my documents are all digital, my higher education course is completely distance learning.

I just sent a request with the horrible image from my webcam, and I'm almost certain it will be denied due to the terrible image quality.

Please fix this, it just doesn't make sense to have a lack of functionality like this.

**Comment by catunderrain** [comment]
Hi there,

I understand your frustration with the GitHub Student Developer Pack application process. Here's a quick guide to help you address this issue:

Contact GitHub Support: The best way to address this is to reach out to GitHub support directly. Explain your situation, including the limitations you're facing with the current application process. Provide details about your digital documents and the issues with your webcam. They might offer an alternative solution or accept your documents via email.

Improve Image Quality: If you have any access to a different device with a better camera, try using that. Alternatively, if you have friends or family with a better camera, see if you can use theirs to take a clearer image of your documents.

Converting Digital Documents: If your documents are digital, consider using a tool to convert them to a format that might be more acceptable for the application. For instance, you can take a screenshot of your digital document or convert it to a PDF and then take a screenshot of the PDF if needed.

Application Feedback: In your application, mention that your course is entirely distance learning and that all your documents are digital. Provide as much context as possible about the quality of the image and the constraints you're facing.

Resubmit If Needed: If your current application is denied due to the image quality, you can resubmit your application with an improved image or after addressing the concerns with GitHub support.

For future reference, you might want to suggest to GitHub that they allow direct uploads of digital documents or provide more flexibility in their verification process, especially given the rise of digital and distance learning programs.

**Comment by Jadiker** [comment]
I fully agree with this concern. I have an HD webcam and an iPhone and it's still not picking up the text. There's no option to upload, and no way to contact support. (Attempting to add a ticket redirects back to this forum with a message that they won't address individual concerns.)
![image](https://github.com/user-attachments/assets/6a2f3dcf-49ad-4f71-bee0-45e0e67c49ac)


Even [the FAQ references an upload button](https://github.com/orgs/community/discussions/111352), but that button does not seem to exist.

The FAQ page:
![image](https://github.com/user-attachments/assets/adaf7a66-d01f-4691-a070-07aa84c09185)

The submission page:
![image](https://github.com/user-attachments/assets/4c563671-7282-46cf-98f2-faa4c24bf87b)


**How should we apply if the system is merely unable to read the text?**
